client/components/tinymce/plugins/wpcom-track-paste/plugin.js

```diff
@@ -37,10 +36,14 @@ function trackPaste( editor ) {
 	const recordPasteEvent = ( mode, types ) => {
 		debug( 'track paste event' );
 		const typesAsArray = Array.from( types );
+		const source = getSource( typesAsArray );
+
+		store.dispatch( pasteEvent( source ) );
```

Comment:
Not mandatory, but a suggestion: since there are now two simultaneous action dispatches here but one of them is just for analytics, consider using [`withAnalytics`](https://github.com/Automattic/wp-calypso/tree/e640615531d1836fa750c5c13f7835677a445bde/client/state/analytics#examples):

```js
dispatch( withAnalytics(
    recordTracksEvent( 'calypso_editor_content_paste', { … } ),
    pastEvent( source )
) );
```